app/jobs/gpo_reminder_job.rb

```diff
@@ -0,0 +1,16 @@
+class GpoReminderJob < ApplicationJob
+  queue_as :low
+
+  # Send email reminders to people with USPS proofing letters whose
+  # letters were sent a while ago, and haven't yet entered their code
+  def perform(cutoff_time_for_sending_reminders)
+    GpoReminderSender.new(analytics).
```

Comment:
how come we pass in the analytics instance? There's nothing "special" about creating it inside the job class, not even a user ID

Reply:
Makes for cleaner tests; the `GpoReminderSender` class doesn't know about the global analytics, so there's no need to mock it.

Reply:
We have a pattern of mocking analytics throughout the test suite. Are you trying to establish a new pattern? Otherwise I think it's clearer to follow the existing one.

Reply:
Changed.

Reply:
Changed.